scripts/zones/Halvung/mobs/Copper_Borer.lua

```diff
@@ -4,6 +4,20 @@
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+    mob:addStatusEffect(xi.effect.BLAZE_SPIKES, 10, 0, 0)
+end
+
+entity.onMobSpawn = function(mob)
+    -- Mob needs to be in the correct state in order to use thermal pulse
+    mob:setAnimationSub(0)
+end
+
+entity.onAdditionalEffect = function(mob, target, damage)
+    return xi.mob.onAddEffect(mob, target, damage, xi.mob.ae.ENFIRE, { power = math.random(3, 7) })
```

Comment:
what lv and gear were you? really don't want to add more randoming our resistance checking should provide variance.

Looking at the video I see the same problem as the spikes damage I commented on, the bat is taking damage in the 30's. its a little hard to catch on screwtube:
![image](https://github.com/LandSandBoat/server/assets/6871475/9c63e30d-4966-4ade-8245-1c8424441bda)

The circumstances of this battle were not controlled for a capture. This is why linking a 3rd parties video in youtube is almost always a waste of dev time here.

Reply:
additional inner workings info on how that power input gets used:
https://github.com/LandSandBoat/server/blob/base/scripts/globals/mobs.lua#L514

you will not get the exact damage of your input back, so you can't just random roll between lowest and highest points you see.